packages/appium/test/plugin-e2e-specs.js

```diff
@@ -8,26 +8,31 @@ import { remote as wdio } from 'webdriverio';
 import axios from 'axios';
 import { main as appiumServer } from '../lib/main';
 import { DEFAULT_APPIUM_HOME, INSTALL_TYPE_LOCAL, DRIVER_TYPE, PLUGIN_TYPE } from '../lib/extension-config';
-import { W3C_PREFIXED_CAPS, TEST_HOST, TEST_PORT } from './helpers';
+import { W3C_PREFIXED_CAPS, TEST_HOST, getTestPort, PROJECT_ROOT } from './helpers';
 import { runExtensionCommand } from '../lib/cli/extension';
 
 chai.should();
 chai.use(chaiAsPromised);
-const FAKE_PLUGIN_DIR = path.resolve(__dirname, '..', '..', 'node_modules', '@appium', 'fake-plugin');
-const FAKE_DRIVER_DIR = path.resolve(__dirname, '..', '..', 'node_modules', 'appium-fake-driver');
-const TEST_SERVER = `http://${TEST_HOST}:${TEST_PORT}`;
+
+const FAKE_PLUGIN_DIR = path.join(PROJECT_ROOT, 'node_modules', '@appium', 'fake-plugin');
+const FAKE_DRIVER_DIR = path.join(PROJECT_ROOT, 'packages', 'fake-driver');
+let TEST_SERVER;
```

Comment:
I would prefer to define variables closer to the scope where they are used (`describe` in such case)

Reply:
ok